src/screens/ChannelListScreen/elements/CustomPreviewMessage.js

```diff
@@ -16,8 +15,8 @@ const PreviewMessage = (props) => {
   const {channel} = props;
   if (channel?.data?.channel_type === 2 || channel?.data?.channel_type === 3)
     return (
-      <Text numberOfLines={1} style={[styles.message, {color: COLORS.gray7}]}>
-        <Text style={[{color: COLORS.gray7}]}>
+      <Text numberOfLines={1} style={[styles.message, {color: '#7A7A7A'}]}>
+        <Text style={[{color: '#7A7A7A'}]}>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The hardcoded color value `#7A7A7A` is used here instead of a variable from a centralized color scheme. This can make maintaining consistency across the app more difficult as changes to the color scheme would require updates in multiple places. Consider using a centralized color scheme.

```diff
- <Text numberOfLines={1} style={[styles.message, {color: '#7A7A7A'}]}>
-   <Text style={[{color: '#7A7A7A'}]}>
+ <Text numberOfLines={1} style={[styles.message, {color: colors.gray7}]}>
+   <Text style={[{color: colors.gray7}]}>
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->